diff --git a/MODULE.bazel b/MODULE.bazel
index dfe0fc7..fdf8b9d 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -5,4 +5,4 @@ module(
 
 bazel_dep(name = "rules_nixpkgs_core", version = "0.13.0")
 bazel_dep(name = "bazel_skylib", version = "1.0.3")
-bazel_dep(name = "rules_cc", version = "0.0.1")
+bazel_dep(name = "rules_cc", version = "0.2.16")
diff --git a/cc.bzl b/cc.bzl
index 5c28f41..fdbd46c 100644
--- a/cc.bzl
+++ b/cc.bzl
@@ -22,13 +22,11 @@ using `nixpkgs_cc_configure(..., cc_lang = "cuda")` or similar.
 
 load("@bazel_skylib//lib:sets.bzl", "sets")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
-load(
-    "@rules_nixpkgs_core//private/cc_toolchain:lib_cc_configure.bzl",
-    "get_cpu_value",
-    "get_starlark_list",
-    "write_builtin_include_directory_paths",
+load("@rules_cc//cc/private/toolchain:lib_cc_configure.bzl",
+     "get_cpu_value",
+     "get_starlark_list",
+     "write_builtin_include_directory_paths",
 )
-
 load("@rules_nixpkgs_core//:nixpkgs.bzl", "nixpkgs_package")
 load(
     "@rules_nixpkgs_core//:util.bzl",
